### ISSUE TITLE
CSS is not generated.

### ISSUE THREAD

**Issue by seb-jean** [SymfonyCasts/tailwind-bundle#5]
I have in template:
```twig
<link rel="stylesheet" href="{{ asset('styles/app.css') }}">
```

My source code is:
![Capture5](https://github.com/SymfonyCasts/tailwind-bundle/assets/12116264/12ea38aa-3dfe-4024-bf18-df002ad2231b)

But it just makes me:

```CSS
@tailwind base;
@tailwind components;
@tailwind utilities;
```

I do have `/var/tailwind/tailwind.built.css` which is generated with the correct CSS.

> Finally, when the contents of assets/styles/app.css is requested, the bundle swaps the contents of that file with the contents of var/tailwind/tailwind.built.css. Nice!

I think this sentence does not work for me.

**Comment by weaverryan** [comment]
Hey @seb-jean!

Interesting and thanks for continuing to test this young code :). Let's see if we can debug. I agree with your conclusions - the content is not being "swapped" as advertised. This is the class that's responsible for doing the swapping: https://github.com/SymfonyCasts/tailwind-bundle/blob/main/src/AssetMapper/TailwindCssAssetCompiler.php

Is the `supports()` method being called? Try adding a `dd()` in `supports()`. Then do a `bin/console cache:clear` and `bin/console debug:asset` to hit that (the `cache:clear` is needed because assets are cached - so that caching will make it appear that `supports()` isn't being called even if it is).

So that's my main question: is `supports()` being called? And if so, is it returning `true` in the correct situations? Perhaps the paths being compared here - https://github.com/SymfonyCasts/tailwind-bundle/blob/main/src/AssetMapper/TailwindCssAssetCompiler.php#L28 - aren't matching on Windows. If so, I'd love to see what those paths look like when `supports()` is being called for `assets/styles/app.css`.

Thanks!

**Comment by seb-jean** [comment]
`supports()` is called but the condition is never `true`.

Here are the values of the variables of the supports function:

`$asset->sourcePath` equal `C:\wamp64\www\zzz\assets\styles\app.css`
`$this->tailwindBuilder->getInputCssPath()` equal `C:\wamp64\www\zzz/assets/styles/app.css`

**Comment by weaverryan** [comment]
Yup - silly slashes 🙄 - thank you :). I've got the bug cornered in CI now on #6

**Comment by weaverryan** [comment]
Ok, should all be fixed up now - try 0.1.2.

Thanks!

**Comment by seb-jean** [comment]
It works.
Thanks Super Ryan 🦸‍♂️!
